core/server/api/shared/serializers/input/all.js

```diff
@@ -1,17 +1,8 @@
 const debug = require('ghost-ignition').debug('api:shared:serializers:input:all');
```

Comment:
This changeset could be extracted and merged as a separate PR. But for now leaving it here for discussion and context